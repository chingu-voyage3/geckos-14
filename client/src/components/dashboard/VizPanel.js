/** VizPanel Component :
The VizPanel displays  a controlPanel and a VizGrid
ControPanel needs the params, actions and parent props
VizGrid just needs the list of vizs
 */

import React from 'react';
import VizGrid from './VizGrid';
import VizControlPanel from './VizControlPanel';

const VizPanel = props => {
  // console.log(props);
  return (
    <div className="viz-panel">
      <VizControlPanel {...props} />
      <VizGrid {...props} />
    </div>
  );
};

export default VizPanel;
