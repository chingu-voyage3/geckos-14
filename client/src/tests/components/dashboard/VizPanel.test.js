import React from 'react';
import VizPanel from '../../../components/dashboard/VizPanel';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const div = document.createElement('div');
  const wrVizPaneler = shallow(<VizPanel />);
  expect(toJson(wrVizPaneler)).toMatchSnapshot();
});
