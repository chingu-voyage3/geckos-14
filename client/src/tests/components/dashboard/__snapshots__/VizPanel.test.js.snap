// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="viz-panel"
>
  <VizControlPanel />
  <VizGrid />
</div>
`;
